src/variables.js

```diff
@@ -135,9 +135,14 @@ module.exports = {
 			const newValue = player[key]
 			const variable = this.variableDefinitions.find((variable) => key === variable.storeId)
 
-			if (variable) {
+			if (variable?.storeId === 'clipID') {
+				const newValueClipID = this.getClipID(player)
+				const oldValue = this.store[variable.storeId]
+				if (newValueClipID !== oldValue) {
+					this.updateVariable(variable.name, newValueClipID)
+				}
+			} else if (variable) {
```

Comment:
Like I mentioned on your previous PR this is intended behavior. I intended to report whatever `clipID` is reported by PlayoutBee. Therefore this constitutes a change in behavior (i.e. a breaking change) and should not be part of a bug fix.

Reply:
If we report back the number we got from playoutbee, why is +1 added in api.js to the ClipID? If we really want to report back what we got from playoutbee, then this +1 should be avoided as well. Am I wrong?

At the end we report something wrong to companion, if we have no clips, index of ClipID is 0 even if we don't have any clips and the name and Id of the current clip is most probably wrong.

A bug should be reported to playoutbee. IMHO it should be the following way (from playoutbee API):
- no clips: ClipID = -1
- 1 clip: ClipID = 0
- > 1 clip: ClipID > 0

What do you think?

Reply:
Maybe we should take this discussion about the clip ID handling to a separate issue and handle it after the fix of the "index problem".

I revert the changes and submit only the index bugfix.